auth-server/src/resolvers/mutation/changeEmail.ts

```diff
@@ -6,9 +6,12 @@ import getTokenFromReq from '../../utils/getTokenFromReq'
 import messages from '../../utils/messages'
 import validateEmail from '../../utils/validateEmail'
 
-export default async (_, args, ctx: Context) => {
+interface argsType {
+	email: string
+}
+
+export default async ({ email }: argsType, ctx: Context) => {
```

Comment:
Removing _ is breaking Apollo-Server. Second argument is args. First is null. Sorry, Did not noticed these changes before. I ran tests which worked fine and did users query which is without arguments so working

Reply:
Resolver input type is https://www.apollographql.com/docs/apollo-server/data/data/#resolver-type-signature